Cargo.toml

```diff
@@ -16,15 +16,15 @@ readme = "README.md"
 
 [dependencies.num-bigint]
 optional = true
-version = "0.2.0-git"
+version = "0.2.1"
```

Comment:
Why is this needed?

num-bigint, complex, and rational are all currently in a transition to 0.2, and haven't been published at all yet -- thus the `git` dependency.  But I don't see why this PR would depend on bigint anyway.

Reply:
(If you want your new num-traits implementations in these traits sooner, please follow up with PRs to their 0.1.x branches.)

Reply:
It wasn't compiling for me and I was a bit confused, and then I realised that I had to run `cargo update`. ><